api/nodes/serializers.py

```diff
@@ -124,6 +127,123 @@ def update(self, instance, validated_data):
         return instance
 
 
+class NodeContributorsSerializer(JSONAPISerializer):
+    """ Separate from UserSerializer due to necessity to override almost every field as read only
+    """
+    filterable_fields = frozenset([
+        'fullname',
+        'given_name',
+        'middle_name',
+        'family_name',
+        'id',
+        'bibliographic',
+        'permissions'
+    ])
+    id = ser.CharField(source='_id')
```

Comment:
Per Steve, please add "label" fields to specify how the field will be shown in the browsable API and options requests.

(this is for any field where the display name would be different from the field name- see [similar](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/users/serializers.py#L14) work in UserSerializer)